testdata/project-v4-multigroup/internal/webhook/crew/v1/captain_webhook.go

```diff
@@ -25,16 +25,17 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
```

Comment:
You can probably drop the "+kubebuilder:object:generate=false" markers now. I would assume that deep copy gen is not running / not enabled for these packages

Reply:
That is great!

Reply:
Done we use it only for the legacy path now.
Good catcher,